applications/nublado/values.yaml

```diff
@@ -41,8 +41,34 @@ controller:
   # set in the corresponding Nublado Vault secret.
   slackAlerts: false
 
+
   # Passed as YAML to the lab controller.
   config:
+  # The controller also manages file serving environments
```

Comment:
Comment not correctly indented (and IMO I don't think this comment adds anything, so I'd just drop it).